woocommerce/woocommerce
# Pass color_scheme param to jetpack connection for core profiler

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partially closes #52876 .

This PR adds `color_scheme` parameter to the Jetpack connection URL. The value will be used on the Jetpack connection page to set the color scheme, ensuring a consistent user experience



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Start the core profiler and proceed to the extensions page.
3. Make sure to select Jetpack
4. Continue
5. Once you get to the Jetpack connection page, copy the URL.
6. Confirm the URL has `color_scheme`
 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Pass color_scheme parameter to the Jetpack connection URL.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
